debian/rules

```diff
@@ -28,7 +28,7 @@ include /usr/share/dpkg/default.mk
 #	-DCMAKE_LIBRARY_PATH=$(DEB_HOST_MULTIARCH)
 
 override_dh_auto_configure:
-	dh_auto_configure --
+	dh_auto_configure -- ${CONFIGURE_ARGS}
```

Comment:
You can do the same thing with commands:

    cd sonic-buildimage/src/sonic-swss
    ./configure --enable-debug=yes
    make

Do you feel this one meet your requirement?